hooks/autohook.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
Autohook is MIT licensed which means that we have to copy its license with it. I think we can just embed the license in this file https://github.com/Autohook/Autohook/blob/main/LICENSE.txt

@brettlangdon are there other things we need to do?

Reply:
> @Kyle-Verhoog @brettlangdon update: switched over from `pre-commit` to using `autohook` since it's super simple and supports using our riot commands (i.e. no need to maintain additional versions). Left out `isort` since it's not yet a riot command, but that can be relayed to a future PR 😄 WDYT?

Reply:
What do you mean embed the license? Does that mean copy n paste the license in the autohook file? Not super experienced with licensing 😕